src/h5web/visualizations/index.tsx

```diff
@@ -115,7 +116,11 @@ export const VIS_DEFS: Record<Vis, VisDef> = {
     Toolbar: LineToolbar,
     Container: NxSpectrumContainer,
     supportsEntity: (entity, metadata) => {
-      const group = getNxDataGroup(entity, metadata);
+      if (!isGroup(entity)) {
+        return false;
+      }
+
+      const group = findNxDataGroup(entity, metadata);
```

Comment:
Checking that the entity is a group with `isGroup` before looking for the `NXdata` group is consistent with:

- the `isDataset` calls in the `supportsEntity` functions of the other visualizations;
- the `assertGroup` calls in the spectrum and image vis containers.

Changing the type of the first argument of `getNxDataGroup` also brings more simplification within this function (see below).

Reply:
Note that I have renamed `getNxDataGroup` to `findNxDataGroup` as an attempt to make functions that use assertions stand out a bit more from those that don't (like `getLinkedEntity`).

`find*` functions live at a higher level: they call `get*` functions and assert on the results. My plan is to introduce more `find*` functions, like `findSignalDataset`.